$NetBSD: patch-ac,v 1.3 2011/01/20 13:06:15 pooka Exp $

The Linux nspluginwrapper-1.4.4 binaries seems built USE_ANONYMOUS_SOCKETS
enabled. In that case we have to handle socket path rename for plugin side
in npw_wrappwer.c and libnoanonsocket.so wrapper library.

--- src/npw-viewer.sh.orig	2008-12-14 12:24:19.000000000 +0100
+++ src/npw-viewer.sh
@@ -83,6 +83,13 @@
 NPW_USE_VALGRIND=${NPW_USE_VALGRIND:-no}
 can_use_valgrind="no"
 
+case $OS in
+freebsd|netbsd)
+    # preset LD_PRELOAD is most likely for the native emul
+    unset LD_PRELOAD
+    ;;
+esac
+
 if test "$ARCH" != "$TARGET_ARCH"; then
     case $TARGET_ARCH in
     i386)
